packages/ses/src/module-load.js

```diff
@@ -387,12 +387,12 @@ const preferAsync = (asyncImpl, _syncImpl) => asyncImpl;
  * compartment and the specifier of the module within its own compartment.
  * This graph is then ready to be synchronously linked and executed.
  */
-export const load = async (
+export async function load (
   compartmentPrivateFields,
   moduleAliases,
   compartment,
   moduleSpecifier,
-) => {
+) {
```

Comment:
this could conditionally be an async arrow function on engines that support it (i.e. not Hermes), if really needed

Reply:
see https://github.com/endojs/endo/pull/2334#discussion_r1677085361